patches/react-native+0.68.2.patch

```diff
@@ -34,100 +34,6 @@ index a6826f3..a7b904b 100644
        child.props.onLayout(event);
      }
    };
-diff --git a/node_modules/react-native/Libraries/Lists/FlatList.js b/node_modules/react-native/Libraries/Lists/FlatList.js
```

Comment:
I'm not following this file properly, can you please point to the fix in it?

Reply:
This is all being removed, it was added in a previous commit taken from a PR in the RN repo